### DISCUSSION TITLE
v0.1.0-alpha.9

### DISCUSSION THREAD

**Discussion by alexjustesen** [alexjustesen/speedtest-tracker#63] [Releases]
## What's Changed
* Fix cron not running by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/62

**Full Changelog**: https://github.com/alexjustesen/speedtest-tracker/compare/v0.1.0-alpha.8...v0.1.0-alpha.9

<hr /><em>This discussion was created from the release <a href='https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.1.0-alpha.9'>v0.1.0-alpha.9</a>.</em>

**Comment by benjamin-feron** [comment]
It works hard today !
Good job @alexjustesen

**Reply by alexjustesen** [reply]
When the baby is asleep ya close issues. I appreciate the recognition!

Only thing I ask is you share the project so we can keep those ISPs honest

**Comment by JourneyOver** [comment]
@alexjustesen do you plan on looking into https://github.com/alexjustesen/speedtest-tracker/pull/49#issuecomment-1284917092 specifically the `edit` part?

Also I take it timezone is still broken as well? As the created at date is still showing things in UTC even though I have specifically set the timezone as `America/Chicago`

**Reply by alexjustesen** [reply]
Good catch on the tz, completely forgot about the views.

**Reply by alexjustesen** [reply]
Also can you open an issue for the speed being off and I can look into it?

**Reply by JourneyOver** [reply]
@alexjustesen yea haha, hopefully a quick fix for that (do you want me to make an issue for this as well) I tried the whole
```
      - /etc/timezone:/etc/timezone:ro
      - /etc/localtime:/etc/localtime:ro
```
thing to see if that would make a difference and setting the TZ env variable (but neither made a difference), also tried changing it myself in https://github.com/alexjustesen/speedtest-tracker/blob/a1087ac933c5c364b81ec03874855dd5852635ca/config/app.php#L72 to my timezone and it didn't seem to make a difference either (both dashboard / results and the database file (in both data / created_at) report the time in UTC)..

Also got the issue opened for the speed being off still https://github.com/alexjustesen/speedtest-tracker/issues/69 which hopefully is another quick fix.

**Reply by alexjustesen** [reply]
tz fixed 👍, will be included in the next release `v0.1.0-alpha.10`
